CITATION.bib

```diff
@@ -0,0 +1,126 @@
+@Article{Virtanen2020,
+author={Virtanen, Pauli
+and Gommers, Ralf
+and Oliphant, Travis E.
+and Haberland, Matt
+and Reddy, Tyler
+and Cournapeau, David
+and Burovski, Evgeni
+and Peterson, Pearu
+and Weckesser, Warren
+and Bright, Jonathan
+and van der Walt, St{\'e}fan J.
+and Brett, Matthew
+and Wilson, Joshua
+and Millman, K. Jarrod
+and Mayorov, Nikolay
+and Nelson, Andrew R. J.
+and Jones, Eric
+and Kern, Robert
+and Larson, Eric
+and Carey, C. J.
+and Polat, {\.{I}}lhan
+and Feng, Yu
+and Moore, Eric W.
+and VanderPlas, Jake
+and Laxalde, Denis
+and Perktold, Josef
+and Cimrman, Robert
+and Henriksen, Ian
+and Quintero, E. A.
+and Harris, Charles R.
+and Archibald, Anne M.
+and Ribeiro, Ant{\^o}nio H.
+and Pedregosa, Fabian
+and van Mulbregt, Paul
+and Vijaykumar, Aditya
+and Bardelli, Alessandro Pietro
+and Rothberg, Alex
+and Hilboll, Andreas
+and Kloeckner, Andreas
+and Scopatz, Anthony
+and Lee, Antony
+and Rokem, Ariel
+and Woods, C. Nathan
+and Fulton, Chad
+and Masson, Charles
+and H{\"a}ggstr{\"o}m, Christian
+and Fitzgerald, Clark
+and Nicholson, David A.
+and Hagen, David R.
+and Pasechnik, Dmitrii V.
+and Olivetti, Emanuele
+and Martin, Eric
+and Wieser, Eric
+and Silva, Fabrice
+and Lenders, Felix
+and Wilhelm, Florian
+and Young, G.
+and Price, Gavin A.
+and Ingold, Gert-Ludwig
+and Allen, Gregory E.
+and Lee, Gregory R.
+and Audren, Herv{\'e}
+and Probst, Irvin
+and Dietrich, J{\"o}rg P.
+and Silterra, Jacob
+and Webber, James T.
+and Slavi{\v{c}}, Janko
+and Nothman, Joel
+and Buchner, Johannes
+and Kulick, Johannes
+and Sch{\"o}nberger, Johannes L.
+and de Miranda Cardoso, Jos{\'e} Vin{\'i}cius
+and Reimer, Joscha
+and Harrington, Joseph
+and Rodr{\'i}guez, Juan Luis Cano
+and Nunez-Iglesias, Juan
+and Kuczynski, Justin
+and Tritz, Kevin
+and Thoma, Martin
+and Newville, Matthew
+and K{\"u}mmerer, Matthias
+and Bolingbroke, Maximilian
+and Tartre, Michael
+and Pak, Mikhail
+and Smith, Nathaniel J.
+and Nowaczyk, Nikolai
+and Shebanov, Nikolay
+and Pavlyk, Oleksandr
+and Brodtkorb, Per A.
+and Lee, Perry
+and McGibbon, Robert T.
+and Feldbauer, Roman
+and Lewis, Sam
+and Tygier, Sam
+and Sievert, Scott
+and Vigna, Sebastiano
+and Peterson, Stefan
+and More, Surhud
+and Pudlik, Tadeusz
+and Oshima, Takuya
+and Pingel, Thomas J.
+and Robitaille, Thomas P.
+and Spura, Thomas
+and Jones, Thouis R.
+and Cera, Tim
+and Leslie, Tim
+and Zito, Tiziano
+and Krauss, Tom
+and Upadhyay, Utkarsh
+and Halchenko, Yaroslav O.
+and V{\'a}zquez-Baeza, Yoshiki
+and 1.0 Contributors, SciPy},
```

Comment:
Hmm, this author list is different (longer?) than the one in our standard BibTeX entry here: https://www.scipy.org/citing.html

Since we already went through the process of reviewing and deciding upon that version, perhaps we should just duplicate it here?

Reply:
Some of the names in this list are not even in the published paper main author list?

Reply:
Thanks for having a look. I didn't know about the existence of another bib. Then perfect, I will take this one.

Strange that there are extra names, I converted the file from nature and did 0 editing.

Reply:
@tylerjereddy I updated with the reference found in the doc. Thanks again.